wiz/pipeline.yaml

```diff
@@ -0,0 +1,10 @@
+steps:
+  - label: "Scan different Directory"
+    command: <command here>
```

Comment:
Do you need a command here? I'm assuming the wiz plugin essentially is the command in this instance.

Reply:
no, we can removed it as is only one required parameter is `scan-type`.